docs/modules/ROOT/pages/csharp.adoc

```diff
@@ -6,21 +6,41 @@ Couchbase
 Getting Started
 ~~~~~~~~~~~~~~~
 
-* Add `http://mobile.nuget.couchbase.com/nuget/Developer/` to your Nuget
-package sources and expect a new build approximately every 2 weeks!
+UWP project
+^^^^^^^^^^^
+
+Create or open an existing Virtual Studio project and install Couchbase Lite using the following method.
+
```

Comment:
What is the above section about?

Reply:
It's the equivalent to the 1.x getting started https://developer.couchbase.com/documentation/mobile/1.5/installation/net/index.html